### ISSUE TITLE
Use `UIAccessibilityPostNotification`s to announce toasts

### ISSUE THREAD

**Issue by BasThomas** [GitHawkApp/Toast#1]
We should announce toasts for VoiceOver using the `UIAccessibilityPostNotification` API.

Reference:
https://developer.apple.com/documentation/uikit/1615194-uiaccessibilitypostnotification
https://developer.apple.com/documentation/uikit/accessibility/uiaccessibility/notification_dictionary_keys
